data.md

```diff
@@ -94,13 +93,21 @@ All data is also available on [our GitHub repository for analysis](https://githu
             <span class="visually-hidden">Expand section for {{ file.name }}</span>
             <img id="icon-{{ date | slugify }}-{{ file.name | slugify }}" src="{{ '/assets/icons/triangle-right.svg' | relative_url }}" alt="Expand section for {{ file.name }}" width="16" height="16">
           </button>
-          <span class="file-title">{{ file.name }}</span>
+          {% assign file-descs = site.data.analysis-files | where: 'file', file.name %}
+          {% if file-descs.size > 0 %}
+          {% assign file-desc = file-descs[0] %}
+          {% else %}
+          {% assign file-desc = '' %}
+          {% endif %}
+          <span class="file-title">{{ file-desc.name | default: file.name }}</span>
+          <span class="file-name">({{ file-desc.file }})</span>
```

Comment:
Right now, when a file does not have a description-name, it this will result in empty parenthesis: 

![image](https://github.com/user-attachments/assets/e1fc279d-2340-4289-acef-99b8abc3b75c)

Reply:
Good catch! Will replace with `file.name` which always presents
```suggestion
          {% assign file-desc = '' %}
          {% endif %}
          <span class="file-title">{{ file-desc.name | default: file.name }}</span>
          <span class="file-name">({{ file.name }})</span>
```